packages/expo-updates/src/ExpoUpdatesModule.types.ts

```diff
@@ -0,0 +1,68 @@
+import { ProxyNativeModule } from 'expo-modules-core';
+
+import {
+  Manifest,
+  UpdateCheckResultAvailable,
+  UpdateCheckResultNotAvailable,
+  UpdateCheckResultRollBack,
+  UpdateFetchResultRollBackToEmbedded,
+  UpdateFetchResultFailure,
+  UpdateFetchResultSuccess,
+  UpdatesLogEntry,
+  UpdatesNativeStateMachineContext,
+} from './Updates.types';
+
+export interface ExpoUpdatesModule
+  extends Pick<ProxyNativeModule, 'addListener' | 'removeListeners'> {
+  isEmergencyLaunch?: boolean;
+  isEmbeddedLaunch: boolean;
+  isEnabled: boolean;
+  isUsingEmbeddedAssets?: boolean;
+  /**
+   * Can be empty string
+   */
+  runtimeVersion: string;
+  checkAutomatically: string;
+  /**
+   * Can be empty string
+   */
+  channel: string;
+  shouldDeferToNativeForAPIMethodAvailabilityInDevelopment: boolean;
+  updateId?: string;
+  commitTime?: string;
+  /**
+   * Android
+   */
+  manifestString?: string;
+  /**
+   * iOS
+   */
+  manifest?: Manifest;
+  localAssets?: { [assetKey: string]: string };
```

Comment:
```suggestion
  localAssets?: Record<string, string>;
```

TypeDoc doesn't like intrinsics keys, Records are preferred.